src/main/java/com/ncc/aif/SeedlingOntologyMapper.java

```diff
@@ -0,0 +1,401 @@
+                    .put("origin", MEMBER_RELIGIOUS_ETHNIC_GROUP)
+                    .put("website", WEBSITE)
+                    .put("religion", MEMBER_RELIGIOUS_ETHNIC_GROUP)
+                    .put("PHYS.Near", LOCATED_NEAR)
+                    .put("PHYS.Located", LOCATED_NEAR)
+                    .put("GEN-AFF.Org-Location", LOCATED_NEAR)
+                    .put("Part-Whole.Subsidiary", PART_WHOLE_SUBSIDIARY)
+                    .build();
+
+    private final ImmutableMap<String, String> RELATION_TYPES_NIST =
+            Maps.toMap(ImmutableList.of(
+                    "GeneralAffiliation.APORA", "GeneralAffiliation.MORE", "GeneralAffiliation.OPRA",
+                    "GeneralAffiliation.OrganizationWebsite", "GeneralAffiliation.PersonAge", "GeneralAffiliation.Sponsorship",
+                    "Measurement.Count",
+                    "OrganizationAffiliation.EmploymentMembership", "OrganizationAffiliation.Founder",
+                    "OrganizationAffiliation.InvestorShareholder", "OrganizationAffiliation.Leadership",
+                    "OrganizationAffiliation.Ownership", "OrganizationAffiliation.StudentAlum",
+                    "PartWhole.Membership", "PartWhole.Subsidiary",
+                    "PersonalSocial.Business", "PersonalSocial.Family", "PersonalSocial.RoleTitle",
+                    "PersonalSocial.Unspecified",
+                    "Physical.LocatedNear", "Physical.OrganizationHeadquarter", "Physical.OrganizationLocationOrigin",
+                    "Physical.Resident"), it -> it);
+
+    private final ImmutableMap<String, String> RELATION_TYPES_STRMAP =
+            ImmutableMap.<String, String>builder()
+                    .putAll(Maps.toMap(ImmutableList.of(
+                            // these are the seedling ontology types themselves,
+                            // in case systems provide them directly
+                            "genafl_apora", MEMBER_RELIGIOUS_ETHNIC_GROUP, "genafl_opra", WEBSITE,
+                            "genafl_perage", "genafl_spon", "measurement_count",
+                            MEMBERSHIP, FOUNDER, INVESTOR,
+                            LEADER, "orgafl_own", ALUM,
+                            PART_WHOLE_MEMBER, SUBSIDIARY, PERSONAL_SOCIAL_BUSINESS,
+                            PERSONAL_SOCIAL_FAMILY, PERSONAL_SOCIAL_ROLE, PERSONAL_SOCIAL_UNSPECIFIED,
+                            LOCATED_NEAR, HEADQUARTERS, "phys_orglocorig",
+                            RESIDENT), it -> it)) // the types in the list don't need special treatment
+                    .putAll(RELATION_SPECIAL_CASES)
+                    .putAll(RELATION_TYPES_NIST)
+                    .build();
+
+    private final ImmutableMap<String, Resource> RELATION_TYPES =
+            Maps.toMap(RELATION_TYPES_STRMAP.keySet(),
+                    it -> ResourceFactory.createResource(NAMESPACE_STATIC + RELATION_TYPES_STRMAP.get(it)));
+
+    public String NAMESPACE = NAMESPACE_STATIC;
```

Comment:
In the original Kotlin, the OntologyMapper had a NAMESPACE val that was overridden (and sometimes used) in its implementations, like RPISeedlingOntologyMapper.  So I got the impression that they should all have a NAMESPACE.  PassThroughOntologyMapper (which wasn't converted because it's only used by Coldstart) took the NAMESPACE as a parameter so it could be overridden.  Maybe RPISeedlingOntologyMapper.NAMESPACE should be set to SeedlingOntologyMapper.NAMESPACE instead of NAMESPACE_STATIC.  Thoughts?  We can take this offline if you like.

Reply:
Do you see a need for it in the Java code? If not, I say we remove it. Feel free to contact me on teams if you'd like to discuss further

Reply:
I think the design is that all ontology mappings (anything which implements OntologyMapper) has a namespace that can be accessed publicly.  In Kotlin, this is specified as a NAMESPACE val in the interface, which is set/overridden in the constructors.  We kind of “lost” this in the Java version because a variable can’t be specified in the interface, so the NAMESPACE String looks useless in SeedlingOntologyMapper.  Maybe the best way to translate this is to add a getNamespace method in OntologyMapper that returns a String.  Implementors can decide if they want to have a constructor that takes a namespace (and sets a member variable, returned by getNamespace) or whether it’s simply a class constant (also returned by getNamespace), but the important thing is that the contract specifies that there’s a way to get the namespace of the ontology mapping.  WDYT?

Reply:
I'm hoping our future mappers will be complete and thus will never have to expose their namespace variable. With that in mind, it's probably best to just leave it the way you have it and deprecate these mappers when we have a solution that can generate mappings from ontology files.